package com.igexin.c.a.b;

/* JADX INFO: compiled from: SearchBox */
/* JADX INFO: loaded from: classes8.dex */
public abstract class f extends com.igexin.c.a.d.f {
    protected static final int e = -2048;
    public String b;
    public d c;
    public Object d;

    private f(int i, d dVar) {
        this(i, null, dVar);
    }

    private static String a(String str) {
        String[] strArrA = g.a(str);
        StringBuilder sb = new StringBuilder();
        if (!strArrA[0].equals("")) {
            sb.append(strArrA[0]);
            sb.append("://");
        }
        if (!strArrA[1].equals("")) {
            sb.append(strArrA[1]);
        }
        if (!strArrA[2].equals("")) {
            sb.append(':');
            sb.append(strArrA[2]);
        }
        if (!strArrA[3].equals("")) {
            sb.append(strArrA[3]);
            if (!strArrA[3].equals("/")) {
                sb.append('/');
            }
        }
        if (!strArrA[4].equals("")) {
            sb.append(strArrA[4]);
        }
        if (!strArrA[5].equals("")) {
            sb.append('?');
            sb.append(strArrA[5]);
        }
        return sb.toString();
    }

    public f(int i, String str, d dVar) {
        super(i);
        if (str != null) {
            String[] strArrA = g.a(str);
            StringBuilder sb = new StringBuilder();
            if (!strArrA[0].equals("")) {
                sb.append(strArrA[0]);
                sb.append("://");
            }
            if (!strArrA[1].equals("")) {
                sb.append(strArrA[1]);
            }
            if (!strArrA[2].equals("")) {
                sb.append(':');
                sb.append(strArrA[2]);
            }
            if (!strArrA[3].equals("")) {
                sb.append(strArrA[3]);
                if (!strArrA[3].equals("/")) {
                    sb.append('/');
                }
            }
            if (!strArrA[4].equals("")) {
                sb.append(strArrA[4]);
            }
            if (!strArrA[5].equals("")) {
                sb.append('?');
                sb.append(strArrA[5]);
            }
            this.b = sb.toString();
        }
        this.c = dVar;
    }

    @Override // com.igexin.c.a.d.f, com.igexin.c.a.d.a.a
    public void a() {
        d dVar = this.c;
        if (dVar != null) {
            dVar.b();
        }
        super.a();
    }

    public f(String str, d dVar) {
        this(0, str, dVar);
    }

    private void a(f fVar) {
        super.a((com.igexin.c.a.d.f) fVar);
        this.b = fVar.b;
        this.c = fVar.c;
    }
}
